### ISSUE TITLE
Multiple rpm/deb packages with package dependency based on project dependency

### ISSUE THREAD

**Issue by zsedem** [sbt/sbt-native-packager#956]
This is a feature request basically.
My use-case is, that i have a multi-project layout named example with these projects

```
example  (the aggregated root)
example-common
example-core
example-web
```

Now we have an rpm package with `rpm:packagBin` command (works like a charm) for the example-core project.

## Expected behaviour

What i want is an rpm package for example-web project, which rpm package depends on core and not contains any jar which is a dependency of core.

So basically i would have 2 RPM-s
example-core-1.0.rpm
example-web-1.0.rpm
and the example-web does depend on the example-core.rpm 

Is this result possible somehow now?
Where to begin if I want to contribute?

I am using systemd, rpm, and sbt-native-packager-1.2.0-M5

**Comment by muuki88** [comment]
Hi @zsedem 

Thanks for your feature request. You are not the first one asking for this :wink: 
There has been a lot of this discussion on this in #723 

I suggest you start there and see if this helps you getting started. 

I would close this issue and do all further stuff in #723.
